public/app/partials/launch/launch.js

```diff
@@ -277,43 +277,56 @@ app.controller('LaunchCtrl', ['$scope', '$rootScope', '$routeParams', '$filter',
         hotkeys.add({ combo: 'k', callback: $scope.prevItem });
         hotkeys.add({ combo: 'left', callback: $scope.prevItem });
 
-        $scope.tableParams = new ngTableParams({
-            page: 1,
-            count: 25,
-            sorting: {
-                duration: 'desc'
-            }
-        }, {
-            total: 0,
-            getData: function ($defer, params) {
-                var ordering;
+        var create_table_attempt = 0;
 
-                for (var prop in params.sorting()) {
-                    ordering = prop;
-                    if (params.sorting()[prop] !== 'asc') {
-                        ordering = '-' + prop;
-                    }
-                    break;
+        $rootScope.getProfile().then(function(profile) {
+            $scope.tableParams = new ngTableParams({
```

Comment:
All inner table creation need to move to separate function. Then do only $rootScope.getProfile().then(function(profile){ return createTable(profile);});